xl-v1-object_test.go

```diff
@@ -0,0 +1,146 @@
+/*
+ * Minio Cloud Storage, (C) 2016 Minio, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package main
+
+import (
+	"bytes"
+	"crypto/md5"
+	"encoding/hex"
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func initializeObjLayer() (objLayer ObjectLayer, err error) {
```

Comment:
The functionalities [here](https://github.com/minio/minio/blob/master/test-utils_test.go#L330) can be used.  Here is a sample implementation [here](https://github.com/minio/minio/blob/master/object-api-listobjects_test.go)